rasa/importers/multi_project.py

```diff
@@ -163,11 +167,45 @@ def _is_in_imported_paths(self, path) -> bool:
     def add_import(self, path: Text) -> None:
         self._imports.append(path)
 
+    def _collect_paths_for_each_property(self, domains: List[Domain]) -> Dict:
+        merges = {
+            "actions": {},
+            "entities": {},
+            "forms": {},
+            "intents": {},
+            "slots": {},
+        }
+        domain_dicts = [domain.as_dict() for domain in domains]
+        for i, domain_dict in enumerate(domain_dicts):
+            domain_path = self._domain_paths[i]
+            for k, v in merges.items():
```

Comment:
optional suggestion: 
```suggestion
            for domain_section, property_path_mappings in merges.items():
```